test/RiskManagerV1.test.js

```diff
@@ -279,6 +320,12 @@ describe("RiskManagerV1", () => {
                   .withArgs(depositStub.address, notifier.address)
               })
 
+              it("should reward the notifier with asset pool shares", async () => {
+                await expect(notifyLiquidationTx)
+                  .to.emit(coveragePoolStub, "AssetPoolSharesGranted")
+                  .withArgs(notifier.address, liquidationNotifierRewardAmount)
+              })
```

Comment:
Wouldn't it be better to keep only one instance of this test under
```
context("when deposit is in liquidation state", () => {
  context("when deposit is at the bond auction threshold", () => {
```
? The logic of withdrawing does not change depending on the surplus etc.

Reply:
I wish we can do that but unfortunatelly we need `notifyLiquidationTx` which is not available here.